rpcserver.go

```diff
@@ -294,11 +294,21 @@ func (r *rpcServer) DebugLevel(ctx context.Context,
 func (r *rpcServer) MintAsset(ctx context.Context,
 	req *tarorpc.MintAssetRequest) (*tarorpc.MintAssetResponse, error) {
 
+	groupKey := &btcec.PublicKey{}
+	if req.EnableEmission && len(req.GroupKey) != 0 {
```

Comment:
Would be nice with an explaining comment here as well.

Reply:
Actually I think here we should also err out early if a group key is set but emission is disabled (catch a user mistake when trying to mint with group key)